### PR TITLE
KFSPTS-6602 reskinning recurring DV tags

### PR DESCRIPTION
**Pull request by jhulslander**

### CODE REVIEW

File: src/main/webapp/WEB-INF/tags/fp/disbursementVoucherBase.tag [hunk 1]
```diff
@@ -1,17 +1,20 @@
...
- You may obtain a copy of the License at
- 
- http://www.opensource.org/licenses/ecl2.php
- 
- Unless required by applicable law or agreed to in writing, software
- distributed under the License is distributed on an "AS IS" BASIS,
- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- See the License for the specific language governing permissions and
- limitations under the License.
+   - The Kuali Financial System, a comprehensive financial management system for higher education.
```
Comment (bhutchinson) [inline src/main/webapp/WEB-INF/tags/fp/disbursementVoucherBase.tag]: Given that this is a custom CU file, I don't think we need any license header.

File: src/main/webapp/WEB-INF/tags/fp/recurringDVDetails.tag [hunk 2]
```diff
@@ -1,30 +1,48 @@
+<%--
```
Comment (bhutchinson) [inline src/main/webapp/WEB-INF/tags/fp/recurringDVDetails.tag]: Given that this is a custom CU file, I don't think we need any license header.

File: src/main/webapp/WEB-INF/tags/fp/recurringDVPDPStatus.tag [hunk 3]
```diff
@@ -1,17 +1,20 @@
 <%--
- Copyright 2007-2009 The Kuali Foundation
```
Comment (bhutchinson) [inline src/main/webapp/WEB-INF/tags/fp/recurringDVPDPStatus.tag]: Given that this is a custom CU file, I don't think we need any license header.

File: src/main/webapp/WEB-INF/tags/fp/recurringDVDetails.tag [hunk 4]
```diff
@@ -1,30 +1,48 @@
...
+			<tr class="header">
 				<kul:htmlAttributeHeaderCell attributeEntry="${recurringDVAttributes.dvCheckDate}" hideRequiredAsterisk="true" scope="col"/>
 				<kul:htmlAttributeHeaderCell attributeEntry="${recurringDVAttributes.dvCheckAmount}" hideRequiredAsterisk="true" scope="col"/>
 				<kul:htmlAttributeHeaderCell attributeEntry="${recurringDVAttributes.dvCheckStub}" scope="col"/>
 				<kul:htmlAttributeHeaderCell attributeEntry="${recurringDVAttributes.dvDocumentNumber}" hideRequiredAsterisk="true" scope="col"/>
 			</tr>
 			<logic:iterate indexId="ctr" name="KualiForm" property="document.recurringDisbursementVoucherDetails" id="currentDetail">
 				<tr>
-					<td>
+					<td class="datacell">
```
Comment (bhutchinson) [inline src/main/webapp/WEB-INF/tags/fp/recurringDVDetails.tag]: You might want to consider adding this class to the <tr> inside the <logic:iterate> tag to add the every other row shading effect:

class="${ctr % 2 == 0 ? "highlight" : ""}"

File: src/main/webapp/WEB-INF/tags/fp/recurringDVPDPStatus.tag [hunk 5]
```diff
@@ -35,50 +37,53 @@
 			</tr>
 			<logic:iterate indexId="ctr" name="KualiForm" property="pdpStatuses" id="currentDetail">
 				<tr>
-					<td class="datacell"><kul:htmlControlAttribute
```
Comment (bhutchinson) [inline src/main/webapp/WEB-INF/tags/fp/recurringDVPDPStatus.tag]: another place you might want to add the class to do the every other row shading

**Comment by jhulslander** [comment]
retest this please

**Comment by jhulslander** [comment]
This is ready for review again.  I made the requested changes (I think), but they aren't showing up in this PR, if look at the commit, you'll see the changes.  Is there something I am not doing here?

**Pull request merged**